spec/factories/street_art.rb

```diff
@@ -0,0 +1,14 @@
+FactoryBot.define do
+  factory :street_art do
+    association :user
+    latitude { Faker::Address.latitude }
+    longitude { Faker::Address.longitude }
+    address { Faker::Address.street_address }
+    city { Faker::Movies::LordOfTheRings.location }
+    state { Faker::Address.state_abbr }
+    zipcode { Faker::Address.zip }
+    image_urls {[ Faker::Fillmurray.image, Faker::Fillmurray.image, Faker::Fillmurray.image ]}
+    description { Faker::Movies::StarWars.quote }
+    artist_name { Faker::Artist.name }
+  end
+end
```

Comment:
Feel free to make suggestions if these Faker choices offend you and you would rather use some different ones!